site.yml

```diff
@@ -157,6 +157,14 @@
       tags: ['ceph', 'ceph-osd']
   environment: "{{ env_vars|default({}) }}"
 
+- name: ceph update
+  hosts: ceph_monitors:ceph_osds:controller:compute
```

Comment:
Can you clarify why the expansion role needs to run on anything more than just `ceph_osds` ?  Why include monitors, controller, compute?